src/integTest/kotlin/snyk/oss/OssServiceTest.kt

```diff
@@ -172,6 +172,28 @@ class OssServiceTest : LightPlatformTestCase() {
         touchAllFields(cliResult)
     }
 
+    @Test
+    fun testConvertRawCliStringToCliResultWithEmptyRawString() {
+        val cli = getOssService(project)
+
+        val cliResult = cli.convertRawCliStringToCliResult("")
+        assertFalse(cliResult.isSuccessful())
```

Comment:
minor nitpick: we can use hamcrest matchers something like
```
assertThat(cliResult.isSuccessful(), equalTo(false))
```
This would be more readable, wdyt?

Reply:
mmm, for me looks a bit more "massive", but yeah, "mathematically" more readable.
will stay with `assertFalse` for now.